.github/ISSUE_TEMPLATE/test.yml

```diff
@@ -0,0 +1,157 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Flaky test
+description: Report a flaky test
+title: "[Flaky test]: "
+labels: ["bug", "awaiting triage", "P1"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this flaky test report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please describe your issue, including the version of Beam and any other languages or tools being used.
```

Comment:
This description should be different for flakey tests.

Something like.

Please list:
- The test or tests that are flakey.
- When the tests started to act flakey if known.
- Any initial thoughts as to why the tests may be flakey.
- A jenkins link to a failure.